plugin/PlayAssetDelivery/src/test/java/com/google/play/core/godot/assetpacks/utils/PlayAssetDeliveryUtilsTest.java

```diff
@@ -287,4 +288,27 @@ public void convertDictionaryToAssetPackLocations_typeMismatch2() {
     Map<String, AssetPackLocation> testAssetPackLocations =
         PlayAssetDeliveryUtils.convertDictionaryToAssetPackLocations(testDictionary);
   }
+
+  @Test
+  public void convertExceptionToDictionary_regularException() {
+    Exception testException = new Exception("Just testing, don't panic.");
+    Dictionary testDict = PlayAssetDeliveryUtils.convertExceptionToDictionary(testException);
+
+    Dictionary expectedDict = new Dictionary();
+    expectedDict.put("type", Exception.class.getCanonicalName());
+    expectedDict.put("message", testException.getMessage());
+    expectedDict.put("errorCode", -100);
```

Comment:
Rather than using -100, you should use AssetPackErrorCode.INTERNAL_ERROR.

Reply:
Updated